kafka-ui-react-app/src/components/common/SmartTable/TableRow.tsx

```diff
@@ -14,6 +15,12 @@ interface TableRowProps<T, TId extends IdType = never, OT = never> {
   onSelectChange?: (row: T, checked: boolean) => void;
 }
 
+const Td = styled.td<{ maxWidth?: string }>`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: ${(props) => props.maxWidth};
+`;
+
```

Comment:
Please move it to `components/common/table/TableHeaderCell/TableHeaderCell.styled`